package/README.md

```diff
@@ -100,6 +100,29 @@ function Cube() {
 }
 ```
 
+</details>
+<details>
+  <summary>Show Vue (Tresjs) example</summary>
```

Comment:
Can you please add an installation step for cientos inside the Vue dropdown?

Reply:
More than an installation step, Would be better to let them know that they need Cientos for this one, don't you think?